onnx_coreml/_operators_nd.py

```diff
@@ -665,6 +664,8 @@ def _convert_unsqueeze(builder, node, graph, err):
     "Unsqueeze": _convert_unsqueeze
 }
 
+_ONNX_NODE_REGISTRY_ND = ChainMap(__ONNX_NODE_REGISTRY_ND, _ONNX_NODE_REGISTRY)
```

Comment:
We do not want to naively fallback to the rank 5 operators, since their translation functions assume rank 5 CoreML tensors and handle the onnx tensors accordingly. 
It is fine for elementwise ops but something like translation of a function link transpose would be totally different based on the rank. 
For instance in CoreML version 3 or lower, concat was only supported on the channel axis (axis=-3), so falling back to that op would only make sense if effective axis in onnx concat is also -3.

Reply:
Good point. For all the elementwise operators do you think it makes sense to import them from `_opertors.py` as `ReLu` is here or move them into their own module `_operators_ew.py` as register them in both `_ONNX_NODE_REGISTRY` and `_ONNX_NODE_REGISTRY_ND`?

Reply:
Both approaches work, for now we started with the first one (importing them). I guess we can continue with that, and once we finish all the layers, we can see if there are too many imports, we can make a new module!